apps/indexer-coordinator/src/monitor/monitor.service.ts

```diff
@@ -22,9 +23,13 @@ export class MonitorService {
   @Cron(CronExpression.EVERY_5_MINUTES)
   async checkNodeHealth() {
     this.logger.info(`check node health started`);
-    const projects = await this.projectService.getAliveProjects();
+    const projects = await this.projectService.getAllProjects();
     this.logger.info(`projects's length: ${projects.length}`);
     for (const project of projects) {
+      if (project.status == IndexingStatus.NOTINDEXING) {
```

Comment:
User not necessarily mark project not_indexing when they stop it. It has to be something else.
I think you need to split desired status and actual status